msteams-platform/includes/requirements-targeting.md

```diff
@@ -0,0 +1,5 @@
+### Specify host runtime requirements (preview)
+
+To ensure intentional and deliberate customer experiences with your app, you can tailor your app's behavior in applicable Microsoft 365 hosts, or omit it from surfacing in contexts that you're not ready to support. Use the app manifest to specify any [dependency relationships](../m365-apps/specify-runtime-requirements.md#specify-relationships-between-components-of-your-app-elementrelationshipset) across components of your app, and specific [runtime requirements](../m365-apps/specify-runtime-requirements.md#specify-runtime-capability-requirements-for-specific-app-components-requirementset) of your static tabs, message extensions, and bots.
```

Comment:
> To ensure intentional and deliberate customer experiences with your app

@erikadoyle, can we say either _intentional customer experiences_ or _deliberate customer experiences_ as I think both _intentional_ and _deliberate_ mean similarly here. I didn't quite understand what an _intentional customer experience_ from a developer's perspective means. The sentence reads quite fine even without the highlighted phrase.

Reply:
@erikadoyle, we treat app manifest like a Microsoft product/service. So, we follow the same branding guidelines as any other Microsoft product/service, such as avoiding using articles before the words "app manifest" unless we are referring to something that is a part of app manifest.

Reply:
> @erikadoyle, can we say either intentional customer experiences or deliberate customer experiences as I think both intentional and deliberate mean similarly here. I didn't quite understand what an intentional customer experience from a developer's perspective means.

Sorry, missed updating this earlier. How about: *To ensure your app always provides customers with high quality experiences,*

Reply:
That sounds perfect, @erikadoyle.